pkg/odo/cli/preference/set.go

```diff
@@ -65,7 +65,7 @@ func (o *SetOptions) Run() (err error) {
 	if !o.configForceFlag {
 		if isSet := cfg.IsSet(o.paramName); isSet {
 			if !ui.Proceed(fmt.Sprintf("%v is already set. Do you want to override it in the config", o.paramName)) {
-				log.Info("Aborted by the user.")
+				log.Info("Aborted by the user")
```

Comment:
Should this one not use `log.Italic` as well? We did use it for the case where app deletion was aborted.

Reply:
No, anything Italic IMO should only be regarding "what to do next". for example, using `odo push`, etc.